Header header

# глубина, [м]
float64 depth
# скорость по глубине
float64 velocity